Dev Meeting Todos (#301)
pupil-labs/pupil-docs: src/developer/core/network-api.md

'SUB_PORT'  # return the current sub port of the IPC Backbone
```

::: tip
<v-icon large color="info">info_outline</v-icon>
Pupil Service does not support the creation of recordings, i.e. the `R` and `r` commands
do not work with Pupil Service.
:::

See this [Python script](https://github.com/pupil-labs/pupil-helpers/blob/master/python/pupil_remote_control.py)
for a full example interaction with Pupil Remote.

## IPC Backbone

The IPC Backbone grants you realtime access to nearly all data generated by Pupil
Capture and Pupil Service. It uses ZeroMQ's [PUB-SUB](http://zguide.zeromq.org/page:all#Getting-the-Message-Out)
pattern for one-to-many communication.

> If you want to tap into the IPC Backbone you will not only need the IP address but also

Reviewer: ```suggestion
If you want to tap into the IPC Backbone you will need both the IP address and the session's unique port.
```

> the session's unique port. You can request them from `Pupil Remote`:

Reviewer: ```suggestion
 You can request them from `Pupil Remote`:
```
